### PR TITLE
fix(#1319): Fix dropdown width attribute

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 1]
```diff
@@ -73,6 +74,12 @@
       }
   }
 
+  $: if(_inputEl && _options.length) {
+    _inputWidth = `${_inputEl.getBoundingClientRect().width}px`;
+    console.log(_inputWidth);
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:79]: Remove

**Review by ArakTaiRoth — APPROVED** [review]

**Review by vanessatran-ddi — APPROVED** [review]

**Pull request merged**